mindjunkies/home/views.py

```
@@ -62,14 +62,17 @@ def get(self, request):
     highlighted_courses = []
+    print(query)
 
     if query:
         courses = Course.objects.filter(title__icontains=query)
+        print(courses)
         for course in courses:
             highlighted_title = course.title.replace(query, f"<mark>{query}</mark>")
             highlighted_courses.append(
                 {"course": course, "highlighted_title": mark_safe(highlighted_title)}
             )
+            
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)